software/dify.yml

```
@@ -0,0 +1,14 @@
+name: "Dify.ai"
+website_url: "https://dify.ai"
+source_code_url: "https://github.com/langgenius/dify"
+description: "Build, test and deploy LLM applications."
+licenses:
+  - Apache-2.0
+  - Commons-Clause
+platforms:
+  - Python
+  - Docker
+  - K8S
```

```suggestion
platforms:
  - Docker
```

As far as I can see, Docker is the only documented deployment method https://docs.dify.ai/getting-started/install-self-hosted/local-source-code

---

resolved.